ios/i18n-export.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
```

Comment:
Can you include a usage of this script in the /resources/build/l10n/README.md file?
I realize we're not really using the crowdin CLI anymore now that the github integrations is connected (can ignore those parts).

But it'd be nice to have a note somewhere to remind us when to run i10n-export.sh

Reply:
Is it possible (or is it unnecessary) for i18n-export.sh to be run automatically?

Reply:
Does the github thing make use of crowdin.yml?  If so, it's only an assist-tool that doesn't belong as part of any automated process.

Reply:
Yes, the GitHub Integration works from crowdin.yml.